crates/core/tedge/src/cli/disconnect/error.rs

```diff
@@ -11,9 +14,9 @@ pub enum DisconnectBridgeError {
     #[error(transparent)]
     IoError(#[from] std::io::Error),
 
+    #[error("Service operation failed.")]
+    ServiceFailed,
```

Comment:
I feel it can be more specific, say `ServiceStopDisableFailed` something on this line?